pkg/components/linkerd/manifest.go

```diff
@@ -27,4 +36,59 @@ identity:
 {{ .Cert }}
       keyPEM: |
 {{ .Key }}
+
+# Following is from values-ha.yaml which contains the values needed to enable HA mode.
+enablePodAntiAffinity: true
+
+# controller configuration
```

Comment:
```suggestion
# Controller configuration.
```

All comments should be capitalized and ends with dot.

Reply:
It is copied verbatim from values file.